README.md

```diff
@@ -81,7 +81,7 @@ Assisted digital satisfaction surveys:
 
 ## Technical documentation
 
-Frontend is a Ruby on Rails application that renders the citizen-facing part of formats stored in the Content Store. It looks up the passed-in slug in the Content Store.
+Frontend is a Ruby on Rails application and should follow [our Rails app conventions](https://docs.publishing.service.gov.uk/manual/conventions-for-rails-applications.html). It renders the citizen-facing part of formats stored in the Content Store. It looks up the passed-in slug in the Content Store.
```

Comment:
I think the "It renders the..." part onwards can be moved to the top of the README. We should also remove the HTML at the top of the README (part of this work is standardising the README structure, and that HTML is not standard!)

Reply:
👍